batch/batch/driver/main.py

```diff
@@ -303,7 +303,11 @@ def active_instances_only(fun):
     instance_pool = app['inst_pool']
 
     ready_cores = await db.select_and_fetchone(
-        'SELECT * FROM ready_cores;')
+        '''
+SELECT SUM(ready_cores_mcpu) AS ready_cores_mcpu
+FROM ready_cores
+GROUP BY token;
```

Comment:
I don't think you want `GROUP BY token`, you want to sum over token.  You can just leave it off, I think and you'll get one row the sum.